includes/class-wc-install.php

```diff
@@ -303,7 +303,7 @@ public static function install() {
 		self::create_terms();
 		self::create_cron_jobs();
 		self::create_files();
-		self::maybe_enable_setup_wizard();
+		self::create_pages();
```

Comment:
@rodrigoprimo Do you think we should add a check for the db version already being set given

- `create_pages()` - the pages only get created if the corresponding setting is not set.
- `install()` - this is hooked to activation only

> Should we maybe highlight this change on a separate changelog entry to make it more explicit 

Good idea. Can we do that through a second line in the PR description?

Reply:
> Do you think we should add a check for the db version already being set given

I guess this could work. But we probably need to find a way to make the check smart enough so that `create_pages()` run when updating WooCommerce if, for some reason in the future, we decide to add a new default page.

> install() - this is hooked to activation only

I'm sorry if I misunderstand something, but I don't think the above is correct. `WC_Install::install()` is called on the plugin activation hook, but it is also called by `WC_Install::check_version()` when WooCommerce is updated. `WC_Install::install()` is a misleading name. Maybe we should deprecate it and create a new method. `install_or_update()` or something like that.

> Good idea. Can we do that through a second line in the PR description?

I'm not sure, but I don't think so (cc @claudiosanches to confirm). Worst case, please add a second entry to the PR description and let me know. I will make sure to add it manually when creating the changelog for 4.6.0 if we finish this PR in time. Alternatively, we could create a single changelog entry but describe both changes. Not ideal, but I guess it is better than not mentioning the change in behavior.

Reply:
> if, for some reason in the future, we decide to add a new default page.

For now I just added the DB version check. If we do add a page in the future that logic could be added to the `maybe_create_pages`. `create_pages()` can be accessed from WP CLI and the tools page so I left it with current logic.

Reply:
I guess that is ok for now.